src/measure_theory/interval_integral.lean

```diff
@@ -144,31 +144,68 @@ intervals is always empty, so this property is equivalent to `f` being integrabl
 def interval_integrable (f : α → E) (μ : measure α) (a b : α) :=
 integrable_on f (Ioc a b) μ ∧ integrable_on f (Ioc b a) μ
 
-lemma measure_theory.integrable.interval_integrable {f : α → E} {μ : measure α}
-  (hf : integrable f μ) {a b : α} :
+/-- A function is interval integrable with respect to a given measure `μ` on `interval a b` if and
+  only if it is integrable on `Ioc (min a b) (max a b)` with respect to `μ`. This is an equivalent
+  defintion of `interval_integrable`. -/
+lemma interval_integrable_iff {f : α → E} {a b : α} {μ : measure α} :
+  interval_integrable f μ a b ↔ integrable_on f (Ioc (min a b) (max a b)) μ :=
+by cases le_total a b; simp [*, interval_integrable]
```

Comment:
```suggestion
by cases le_total a b; simp [h, interval_integrable]
```

just makes it crystal-clear what's going on

Reply:
otherwise it all seems really clear - haven't seen this section of mathlib before and it was all very understandable